tests/cluster/tests/30-delkeysinslot-repl-to-replica.tcl

```diff
@@ -0,0 +1,76 @@
+# Check delKeysInSlot replicate to replica
+
+source "../tests/includes/init-tests.tcl"
+
+test "Create a 3 nodes cluster" {
+    create_cluster 3 3
+}
+
+test "Cluster is up" {
+    assert_cluster_state ok
+}
+
+test "Cluster is writable" {
+    cluster_write_test 0
+}
+
+test "Instance #3 is a slave" {
+    assert {[RI 3 role] eq {slave}}
+}
+
+test "Instance #3 synced with the master" {
+    wait_for_condition 1000 50 {
+        [RI 3 master_link_status] eq {up}
+    } else {
+        fail "Instance #3 master link status is not up"
+    }
+}
+
+test "Instance #4 is a slave" {
+    assert {[RI 4 role] eq {slave}}
+}
+
+test "Instance #4 synced with the master" {
+    wait_for_condition 1000 50 {
+        [RI 4 master_link_status] eq {up}
+    } else {
+        fail "Instance #4 master link status is not up"
+    }
+}
+
+set randomkey [R 0 randomkey]
+set randomkey_slot [R 0 cluster keyslot $randomkey]
+set slot_keys_num [R 0 cluster countkeysinslot $randomkey_slot]
+
+test "Instance #3 is a slave, have the 'randomkey'" {
```

Comment:
we're trying to avoid the use of t he term "slave" in new code (If we can), and specifically in parts that are visible to users (documentation, logs, and test prints)